packages/core/src/tools/utils/arrayUtils.ts

```diff
@@ -12,3 +12,7 @@ export function removeItem<T>(array: T[], item: T) {
     array.splice(index, 1)
   }
 }
+
+export function includesItem<T>(array: readonly T[], item: unknown): item is T {
+  return array.includes(item as T)
+}
```

Comment:
<!-- Warning represent user-facing problems. If possible, it's great to follow this kind of comment with a suggestion.  -->
**❗ warning:** ‏as raised by `unit-bs`:
```
[IE 11.0 (Windows 10)]
	TypeError: Object doesn't support property or method 'includes'
```
We have a polyfill though: https://github.com/DataDog/browser-sdk/blob/main/packages/core/src/tools/utils/polyfills.ts#L3

Reply:
I missed it, thanks. It's done.